docs/detections/rules-cross-cluster-search.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[rules-cross-cluster-search]]
+== Cross-cluster search and detection rules
```

Comment:
What are your thoughts on changing this to something that's more Security-focused, for example: 

```suggestion
== Detect threats across clusters
```

Another option that's more search-focused is:

```suggestion
== Search for threats across clusters
```

The current title is already clear and descriptive, so these are just suggestions :)